Practice in php/colorcalculator.php

```diff
@@ -0,0 +1,42 @@
+<html>
+
+		<form action= "">
+			<select name='ColorSelector1' class='ColorSelector1'>
+					<option value=''> Select</option>
+					<option value='Red'> Red </option>
+					<option value='Blue'> Blue </option>
+					<option value='Yellow'> Yellow </option>
+			</select>
+			<select name='ColorSelector2'>
+					<option value=''> Select</option>
+					<option value='Red'> Red </option>
+					<option value='Blue'> Blue </option>
+					<option value='Yellow'> Yellow </option>
+			</select>
+			<input type='submit' Value='Result'/>
+
+</html>
+
+<?php
+
+
+if(isset($_REQUEST['ColorSelector1']) && $_REQUEST['ColorSelector1']=='Red' || isset($_REQUEST['ColorSelector2']) && $_REQUEST['ColorSelector2']=='Red'){
+		echo '
+		<body style="background-color:#af2323">';
```

Comment:
Rather than echoing the full body tag each time, you could save a variable called `$backgroundColor` or something like that, and then below all of these if statements say:
```
<body style='background-color:$backgroundColor;'>
```